automation/components-functests.setup.sh

```diff
@@ -99,6 +99,15 @@ git-utils::fetch_component ${component_path} ${component_url} ${component_commit
 
 export TMP_COMPONENT_PATH=${component_path}
 
+# TODO remove
+if [[ COMPONENT=="kubemacpool" ]]; then
```

Comment:
```suggestion
if [[ $COMPONENT == "kubemacpool" ]]; then
```

IIUIC the statement you used would always return true

Reply:
the PR is WIP / draft as i debug something all those will be dropped
not ready to review (saw the failure just after i asked sorry)
thanks

Reply:
It will be dropped before this PR is merged, just poced to see it fix the root problem